datasets/ua/war_sanctions/crawler.py

```diff
@@ -0,0 +1,397 @@
+from zavod import Context, helpers as h
+
+
+LINKS = [
+    {  # child kidnappers
+        "url": "https://war-sanctions.gur.gov.ua/en/kidnappers/persons?page=1&per-page=12",
+        "type": "person",
+    },
+    # {  # child kidnappers
+    #     "url": "https://war-sanctions.gur.gov.ua/en/kidnappers/companies?page=1&per-page=12",
+    #     "type": "company",
+    # },
+    {  # russian athletes
+        "url": "https://war-sanctions.gur.gov.ua/en/sport/persons?page=1&per-page=12",
+        "type": "person",
+    },
+    # {  # ships
+    #     "url": "https://war-sanctions.gur.gov.ua/en/transport/ships?page=1&per-page=12",
+    #     "type": "vessel",
+    # },
+]
+
+# TODO: fix strings that were merged on | but were not supposed to be
+
+
+def lookup_override(context, key, lookup_type):
+    override_res = context.lookup("overrides", key)
+    if not override_res:
+        context.log.warning(f"No override found for {key}")
+        return {}
+
+    extracted = {}
+    for override in override_res.items:
+        if override["prop"] == "name":
+            extracted["name"] = override["value"]
+        elif override["prop"] == "registrationCode":
+            extracted["registrationCode"] = override["value"]
+
+    if not extracted:
+        context.log.warning(f"No matching properties found in overrides for {key}")
+
+    return extracted
+
+
+def extract_label_value_pair(label_elem, value_elem, data):
+    label = label_elem.text_content().strip().replace("\n", " ")
+    value = value_elem.text_content().strip().replace("\n", " ")
+    value = " ".join(value.split())
+    # we use it for splitting some strings at a later stage
+    value = " | ".join(
+        [text.strip() for text in value_elem.itertext() if text.strip()]
+    ).strip()
+    data[label] = value
+
+    return data
+
+
+def crawl_index_page(context: Context, index_page, data_type):
+    index_page = context.fetch_html(index_page, cache_days=3)
+    main_grid = index_page.find('.//div[@id="main-grid"]')
+    if main_grid is not None:
+        for a in main_grid.findall(".//a"):
+            href = [a.get("href")]
+            for link in href:
```

Comment:
for link in href is unnecessary because we know there's exactly one element in href.

But this could be simplified to

```suggestion
        for link in main_grid.findall(".//a/@href"):
```